### ISSUE TITLE
In Combat - Alpha bugs out sometimes

### ISSUE THREAD

**Issue by Nillx** [WeakAuras/WeakAuras2#4093]
### Description

I have my aura set to  stay at 30% alpha when im not in combat but sometimes when i enter combat it stays at 30%. Before that it was working normally and was always switching to 100%. 

![image](https://user-images.githubusercontent.com/34476875/205455874-354a62a5-1043-40b7-bc60-b793d074d4a2.png)

### WeakAuras Version

5.2.2-7-g43ddad9

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Lua Error

No errors.

### Reproduction Steps

Its random, sometimes my aura stays at 30% alpha, sometimes it works normally and stays at 100%. Try entering and exiting combat.

### Last Good Version

5.2.2

### Export String

!WA:2!1M1FSXXr1)C5kKktlnXHVPbAkz1jK)A36Cj(CSbfLkZD235ZPoND37CJDtTUB3BN72jzVDxMzpN7sP)JLafjeQs(FOTerdMFiqce0dbIkHkOOGeiAHb3sBracKfTfO0IicHesGeVzMDVF47sIDLyKU7M9nV5nFM38EV592luQ(R0Vr)gxk(QoRGiwA1PFOtLEI35ddT79DspXB(Q82)kC41XfDSZ6uLueDODrQnxPsuK3jdrQl79dc1qZUOPdzEhSTN(KjZKlP6guxnY5vD808Wo2F3yvjwUhY0ZZLEIJE0lOv2jk25OPh5CpYihXiZdD0yhNOvKZjDCvQNgXBa9IvPEovI0xFRz4Kx(GzcSn2Ba2D4(hTCkQzPGuEafTQeT8i7v6tsI6IkcuFS(uG2zhzzOFKewoogrgwskMGukId1lG0OcslyB6yvpG2ycAzC4YlsFpEFLQAlaOckAwG0cUgAEOb5DZuTIoImKyAeKxvITaeNT1yl3hY2Opu0IvjeKTxwje7qqtJeKXAw4lkuzdo0qS7SGEjyhtnHT8Uz3P5ALCS9szPvMsMBHCZotMKIJM8fDSCiNkm0wJISkjohsmBYu5wvxJmPyW0t8CFqE7KPN4TxM3kMEI36X4Thk86giQgGBahMxvCUjpyxkeO40mU0hyJQuuExKNUMNNfYCn(J2iWMX0TFE)IwAuAEnBJ88nEH1AsRqcIJf6aQvQA5H7BvoDbhjO4lIoa7W9r80SansmE)n7umdOtX2LTqx7Uyh(ax7UkOlzLZzdUGwbzIlcOjHGz2hzpWaRBGlvcxewT6cPdcG4u1L3xviDGVgtLmEU0pyMzMoDoG(gTjRfWMja7qjYkjni59xZDf3wRo5Io2OzmO7EtAfhhpZ5joLjik10DVGSIBCoWwfzGmon22CDcQmiKC1DreUHkCECDHgEQwQCIhQg0f5UVeWH)f1igkQAydLeAe5Pro)XV3zS9qCCHE0ev98aNLh9uvR4UGBCcX5cj0Q65uyDpNIWbdfw0dg6tVUQMLRPw4yvXgnMDqvAM6UN6IpCSn1DigiscTIN3G446E4MR8uGfOtzf(iCvNnqH7hlSW6BDPBihpGFP7(6Wh)0ogi9thpZcXNDvpcUCzafzgG439ns4b6a2D0qkI88NAaE6EOQchGRlP)WAeSMUfIUBvW(cIMKgPz5zUrzKnIGlM105cZzFvQ4N4W5ZkOevHqc6Uq0leztAvDX0MNGkHRPMD(KZol7oQDBlf49oOMLvw(QshwrW5WkKQ2iPB6WkrJgv6hJlPy74P0KBbrptKDGhUhPksH7vZhq8vpIi1IS0Fh8vlHl3g5kAEMWi8FAjiyTfqt5bGaqZpB8LsQMh2jtot8zN5rINBM5YKFY0XZmDYPIi43H0n)IyWZKz68Nzo1zNkIe6cU5TDCKiXmd2TDGp1fYKm)8ZDgynxy(PINlPCTAUuLa0HHLyKHhxXWPj5wQar0(Hvm4(cWIjppurAg1HzbyrfECshhldNlypiEOoKqZJNZIxUzm)2Bfn1SlJmGX4Nxd314CdPB4GU((1CNx(DbE4kOGBrAVHQ5ILGphWbWPylPC)T2tDnLGraMVXmHTnq1aoWDpuPQwwWinvvDWWJ36C2)eZ)uRyuQdrOqPBXGqyDZ97eC0UtslhNH1hQlC4lyTOBrhaXWv03kXoSl2AZ3TQljDYUKtpfbYIIUvYMFk37j3MAY)5H6shg8B7cdOTkC7JLfCBqFYipu3u3ux2H7YHzyLE6ZQw0ev88Shu1wRce)PzOTSvHB4Q1i)KXZMlF2CXvZbxFRdef3XKo5SZNAHzxRQTFC3(AyGPNtCiUckMMDD39Pj6NtoopQD4RJ5xSyRz9WYBoQVMMnUIqJpo79DP9THhrZMAbwe5ebX3iWOvegNqrqGndQU4sI4HL)oDyXnRnNPjB6e2WfNRI0OsXGBj2uaaDFRM2AbEEdlDLw0)3LGWeilpTf9)DPMXQdI4gvCJIW8RtsrLotDycgKRMyja31b75uoIYidDFdEgK25JddtZQTcY4SraLk)khAKLpBZzHnw(SrUa2WZmYYG4AjnxTIqIiTcIcF0PqSpu9WStgQ)6H6VwiPslryDc)6uuORYvtz9GG0L9m3JCu1WYPvlCIkAy7lD3SpryrUwcQMSsHzLdZSdZmlSMGexnVbf0Y4YMEz5uyU7Hnn7oKzOXLzb25yy25Rf65FxQ9F3CThvqwOLBJkrqvMWuWblTZJmrqS5DOyoqfbYoTMDviCv9UJUifkpaAXZgHZBKOrPYd1OrJSyKL7b31VbCV0w4230P2WkTc92viiPixrZQkx7miOhSejoEeUbf5eWDBNHBSm4q3Fr4(xPnc)Y8J1RLsiMHBBS2JlbFyve2s14MtRjozLUQsdbH)1ZiV54wEklMXI()UuqKWU8wceGY9PmOCkGVIC4HgkqA9KXLcyCPoYSGveGRa6C4wB3prpWkzKHvkdF0HpAWhsm4z4Jo8rl2)ZTiLK1lZV7nkKlFu9YBnbqAqkvTNI1wzk10IQYewB8jaMyPM23gtYrIU5irNCWZbOBPae7GlccYbIug1IZiQ(K85RLRgpSPa(D4lPx(ezrEWndqzkcPmOp2GIRhoaNqv1T6pAB9p(Yd1U3lnQi1LUCw9TUAUNeYU1tX64Pr74PJV8n2f0xSPAdWPAdWPAdWPMUBXTfxmNqmdgIrcZ2BkikkMhmLtaIGEDD)sPMXofbks3SHO2OtJHkZif2K6HlE(6t5FRPjXVcTrwhUvxvyErpXbytVb8OEZkVgxsaYGgjj4UxELI5PMAqYWtkRitqrwD2kBiRQlVJO69WbpkF5mXaZ1g(u4vFh(Q(piF5bHcfkSFvH5xbtXqUGfcyhzajkmIY84Ail20RXbPyaDte)IL7ztXEnbuFSbpDIyXNAQTwkSwnZ1fCnzZxvrIl64uj0gqvRa6MZv8YFUYrbfBddeTiblOKGV)C3708Y)aRzWq0dctsxL)IqYc7cvlULSQ4w33d)q5ZXEYgzHCyve3oCc2tRdjTJRwHD55dFSOJVcBADpNYLTqWgOKwvlVcj4lN7H5tIkYOMczEROrlcBhyTueB1OXopQ(AntFMa7oOc(IfsWZoRFf2Ld7V4BkD0cak7PzFE2LFpWrPkpna0gR4xGRi0Fbyi)5PlEhvaEzFbGiatwC2vovy4uH9fDJ67YYHJ44s5c803PbLPbv4azW5OiKruf2xIi6LAA2xPa7RUERzd4mq0PN4foeV9Y(lrBv8VdwIe(lrRz32s4J(JkvjoL4U9cnSu2ypfmOXfL6bswhPaAvHSxtcFWfxi9eIExomOfzFn2xVDTM414TvTwiW6AhO1eYqSLe981ASVzB7Kqn1wn7StuABzf8vASVD7RGaZSNf28sueS5zFNEUZjY3wz3BDPkFBU1LcHJSvLDV5BEPS3MB6UKDV32H832R7dGBX(E1mok8Ru36gp9eVEvE7Z089w3Kch0d0Bfc)D80gWJcyv(gF3g6Hb6TE4gkYBX23FD7y77VND3FqkVC47hQ0pYIplBWdlP4NfPVUHhW7J(keQtjVtJTF0DbXGi8xhcrhxoRhYnCSkA1(PjO8UWIfRc2E23lyJ6lfylE)5ahwTkWw0J7(kIGb(PExabBuKwrtU3ScCFeXiQCD0QnQa(qiXRSNa4Fqzs6IqYkTTvoHCpm)Tns0JngxL8mSRycl7DhWJCgbP4dQs3)VEM8FWscIc2YJf4IiM9i(AJWFvi0VT498Xz)ND7))7L9cCv0ysn1ySxCVaPVKoFwJSiaJ9VOI8Mu(Exe2AKOWYSi7kwVxqySF12qolXLZs9uolXUsOoqAS2q62r4X6niJ5dYF72wo9gKX6jihDhcYr7nih1hKZgA7kNEdYr7jip(oeKhV3G84(GCOTRy6ngpEpX4y7qmowVX4y(y8X2UIP3yCSEIXX3HyC8EJXX9X4lUDftVX44by0pX2GuT3ii37jdYKn0AI)wmk2aPQY)FEK5EkY5AiDzXFx6EyFloVYv)hcxG8D2pVfn9eVX)K3(3qO5Fxi21AX0p(unhl9e)5NJ3E20t8NEsE7tfM9kHy)8WSFZ0SNd47NCqEB10tSzdE7vspXB)F4Txnm7ndX(B)J(yVKj75Bj7VFRU)1qSx)LyFpqi)Er7j6Uty2Fpe71cX(lx72zFJBoJxLNSD6j(1p1t9J21U(yS)WNLTziwJtbK(fhM3Go)YhI3(KHzF5cSFWnvAYQAsls7F02)Rcxm01Bv4sZZH0t8sJo6hFx7A)6uufOMh1yrpw0yj4ho(fiLgByGStKyUCPD3BR)Dj5FfHU8VgQrjcKKDwpO4jT9WUCYRllOjfNk)LCONDs1KjZa2mj43xPt0wbr9uCXLRuxXXgUYQKtvaWI6BlHGYp5DmaPaPQajQazLIjkgOkykuSp2MxCgQqeSXg83vRYr8ZKPHdbd1ZlQGB90ZPoZJmxMCXNDD(7(uCLe9dna7mJZsDODTAqw(RITl6urxZtv8(BcrK)HeaNVwO3p)0E0vDbdxO266St2aBd16jFTRd0a6weLu8kLl4EqCzBOmq)R(futYRSKwqvOgn7NQow0yrJ1)kV8I)3d

**Comment by Nillx** [comment]
Could that specific aura affect my regular ones that have no custom stuff in them?

**Comment by InfusOnWoW** [comment]
No idea, I won't look at the aura.

**Comment by Nillx** [comment]
I made a new basic aura that shows my festring strike wounds and its happening with it as well 

![image](https://user-images.githubusercontent.com/34476875/205749372-cb03988e-d782-4677-89d2-214a9ed9ef5c.png)

![image](https://user-images.githubusercontent.com/34476875/205749483-8b3abef4-84b6-4ffa-b58a-72cbb48480c6.png)

![image](https://user-images.githubusercontent.com/34476875/205749538-5038b524-825b-4f0e-860d-2e08e2ea823f.png)

![image](https://user-images.githubusercontent.com/34476875/205749644-2be7ccca-1751-452f-9d73-1db084a72f4d.png)

`!WA:2!1v1ZVTrXx84Avfjdsn18LO0OcQFReHcsTQXTnvvuPwVP21PKyhw70FCOYE2DNDNPz9mdZmRDCqib(akIJ(afUH8zKqYs8pqupH4WkREMd5s5aId5g34n7AN2qP(G338EZ8(18(8zYuoF78E5927dhsD5S68iPloZyuKMWL1eAkNPYj7vZ3xH1vU99(L1Mb(ncXCb7BYPmTZQLQ2OK9yxop0J3LvVlvGBjMZnsP5TBG3rVLWdPXorjF2F6(k5fGjsKBsiwXwPrsDohFkJQi5SGp6C91sAqawQQUKCI4EV7auKeXqTXQfDw(gx9klFzj4TASWEToqf5G7Gz6nLyF6o213S06RpksHBQHGQPcYqpip89B0tGLvkAVr5TwFOrrDcVBn2(QKpfHCQdwcNRkeMwhACq65wnePuezeljk2AABSN40g7kb2L6tDBgbjoPVhKJMcl)Y2jB1PcgfQj2P59H3NkJcb1NBZquqe(aWd3vYJenD5rmn5O6Ooet6oJAUAX6nAwVrr7g9HafgUMNAXfoXF6APHcX20qkyzcSd0edW6dC5TDOm8giTlbRgAkR15DXk9O0BLMeQho(9g3oDdPvF8zgabpTnNBKhv9KiwsNOakmuCAuICJu7BW9W)0mhcx)yjdfEF4ocQ2V4WTXyrrtVqBBQFYafo0pzmj(KhO0u3T7DNjTgYqOkTXbMR)pE(4lmgw6GC3oaAemV9wu8(Ay2PzYF(CjKQnfnnD8jRYioZRTb1eblgNHhCK5Z)bkXCjRueem7TkpKlVxMmzYEqQ2xMKI3jrdm)Zb)qDFa1ttSkclpmnq0DWEjk)6XPGGhLInYSFI9Na9xQFV4touY1W4Uz(3QATQLe5FLe4Htot8fKqvBmmXCQlNy(SZmmrPBA6MTYT(RF3875zhmPOzAXCLL0Dp3NfH8G2k6CnAi(O3uFZRhm(bJOAIedJeHEzowsnTq(p6RIjcWitI6UCP3dKir)hmriT47qvuNqd4)vkMjn2XhLYjGVAB1y91GUY))nLScjaOmdvzF5jRt3fF9XhRhL5ynUhnTX5qW0acqyz)T)SHWYkKJ82BEjLzgVeZB3oc4dYnWaDnGzjrK3i7Aa4GZ8sa0TgCKUwwsEiE(4hNR)u4EllfKpgvsnYGMlyKp44UyjhfeZq8Zov8JN)zNQLt6wn7mHyQdMqDdXeRKnh7plyWcyft8RDIVaTJUtPInQ8Pvx7UvAa6h6rbEbxOe6z22yJJeyTdsRbxn(v87wuYarhXldQ0pLX1ipmP6aQcKULCxaXa0kNqmxcDNefWzsQhK8KHYeuQ5EZY8ibqIQqAdmgt6pLoV1qWIhnHnFXLSbAf3Txj(uNDM(Dqskcgl6tzPbZUdkmcNr6sqSaGoCP4VCZSx(sf6lKCy0q3ZgfkiiX)BQVnGO7qvgN41Yofou5wVOR53JEDHSoqQ4tdYnYxc8T11gKXSw7Y5TZyzq2KHAUBNuoRfY8ndpp1ZQbWoA31aSRC76)6t)ryOj(RYEy6CvzJFm1Vt9vTlvQAYWYQtYoGlei7WTker9gTXNVZ9xru7ApS8nGHntmGh52z6ZNHV4V)dJJ70YWVkcr9wds1fo9N8BJOS05FiLwAei6IlzEbq1sSanGXL4jpgNOTKuYLQwdqmA6rwj(TkpE6JonnVkab2CHOIVq8n3hJuM2aMfOjZ23SYumX30QnIYkh)DXp1ST4VF24Fa(82)Bn5b6ZRDPRCPLVyHlg4F1RE9cUxpFNN)W)5p`

**Comment by InfusOnWoW** [comment]
I can't reproduce the issue. Did you try to reproduce just with that aura installed?

**Comment by Nillx** [comment]
I have not. Does it work if i disable all my WAs or its better to backup my savedvariable and delete it?

**Comment by InfusOnWoW** [comment]
Empty settins are the easiest, just rename the WeakAuras.lua and WeakAuras.lua.bak file from the WTF folder.

**Comment by Nillx** [comment]
So far im unable to reporoduce it again when renamed the lua file... I really dont know what is causing it, maybe another addon? It was fine in vers 5.2.2 and started happening with the next alphas.

**Comment by InfusOnWoW** [comment]
I don't think there were any changes to the condtion checks or alphas in those patches.

Was the test aura in a luxthos group before?

**Comment by Nillx** [comment]
Im using only 1 aura from Luxthos's auras, everything else is without custom code. This aura was in that group yes but i moved it outside any groups.

**Comment by InfusOnWoW** [comment]
Make sure you have bugsack and buggraber installed first.

**Comment by Nillx** [comment]
I always have them on
